examples/image_classification/main.js

```diff
@@ -133,18 +90,14 @@ const updateScenario = async (camera = false) => {
   predictPath(camera);
 }
 
-const updateSupportedOps = async (backend, prefer) => {
-  supportedOps = getDefaultSupportedOps(backend, prefer);
-};
-
 const updateBackend = async (camera = false) => {
   streaming = false;
   try { utils.deleteAll(); } catch (e) { }
   logConfig();
-  await showProgress('Updating Backend ...');
+  await showProgress('Updating backend ...');
   try {
-    updateSupportedOps(currentBackend, currentPrefer);
```

Comment:
this call to the `updateSupportedOps` is necessary for updating the global variable `supportedOps` (defined in `base.js`). otherwise, nothing will be offloaded to webnn.

Reply:
this method serves the same function as `getOffloadOps` and designed to be called before the `utilsInit ` so that it can update supportedOps based on the selected backend before reconstructing the polyfill model.